config/config.go

```diff
@@ -393,6 +393,12 @@ func InitClient() error {
 			break
 		}
 	}
+	for _, prefix := range prefixes {
+		if val, isSet := os.LookupEnv(prefix + "_TOPOLOGYNAMESPACEURL"); isSet {
```

Comment:
Let's follow the pattern above - prefix + `_TOPOLOGY_NAMESPACE_URL`:
```suggestion
		if val, isSet := os.LookupEnv(prefix + "_TOPOLOGY_NAMESPACE_URL"); isSet {
```

Reply:
Fixed (making note for myself)

Reply:
Yeah -- for anything new, we should use definitely use `viper.Get(...)` as we import the environment variables as well.

This is mostly for backward compatibility: `AutomaticEnv` only works with a single prefix and for namespaces we've supported a list of prefixes.

In other words, we shouldn't create any more of these -- but I'm ok with this one for now.